test_sum_terms.py

```diff
@@ -0,0 +1,3 @@
+Test.assert_equals(series_sum(1), "1.00")
```

Comment:
Where are you getting this `Test` object from? You should make the tests from Code Wars executable as if you were writing tests for your own code.